rsocket/statemachine/StreamsFactory.cpp

```diff
@@ -65,6 +77,10 @@ StreamId StreamsFactory::getNextStreamId() {
   return streamId;
 }
 
+void StreamsFactory::setNextStreamId(StreamId streamId) {
```

Comment:
This method is misleading, it's not setting the next stream id, it's setting the last if anything.